domains/measurements/Measurements.API/Tests/Fixtures/TestServerHelpers/TestServerContext.cs

```diff
@@ -0,0 +1,55 @@
+#region Copyright notice and license
```

Comment:
Why this?

Reply:
Taken from the wallet, which got it from somewhere else. It basically says it's been copy pasted